src/main/java/org/springframework/data/elasticsearch/core/SearchHitsUtil.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * (c) Copyright 2019 sothawo
+ */
+package org.springframework.data.elasticsearch.core;
```

Comment:
Let's have a proper license header and here and move the utility to `org.springframework.data.elasticsearch.support`.